tests/platform_tests/test_platform_info.py

```diff
@@ -360,6 +361,7 @@ def test_thermal_control_psu_absence(duthosts, rand_one_dut_hostname, psu_contro
 
         logging.info('Shutdown first PSU and check thermal control result...')
         all_psu_status = psu_ctrl.get_psu_status()
+        assert all_psu_status and len(all_psu_status) >= 2, 'Failed to get at least 2 PSU status: {}'.format(all_psu_status)
```

Comment:
LGTM. Just one minor comment: 
Can you replace the raw `assert` with wrapper `pytest_assert` defined in `tests.common.helpers.assertions.py`?

Reply:
Sure.

Reply:
If there is not enough PSU len(..) < 2, do you want to fail the test or skip the test.

Maybe pytest_requires() is a better choice than pytest_assert?